crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -1710,6 +1732,22 @@ impl ScheduleBuildSettings {
     }
 }
 
+/// list of [`ComponentId`]'s to ignore when reporting ambiguity conflicts between systems
+#[derive(Resource, Default, Clone)]
```

Comment:
Debug for sure here, but a method for pretty printing would *also* be very useful for external tools.